src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+
+    private Node<K, V>[] nodes;
+    private int capacity;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        capacity = DEFAULT_CAPACITY;
```

Comment:
Why do we need this ?
redaundent initialize

Reply:
Because  when we created a new object of MyHashMap, capacity will be 0;

Reply:
In this implementation table.length = capacity. You don't have to store it in a variable